machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc

```diff
@@ -24,6 +24,12 @@ If you do not have access to the {op-system-first} images that were used to crea
 
 Before you add more compute machines to a cluster that you installed on bare metal infrastructure, you must create {op-system} machines for it to use. You can either use an ISO image or network PXE booting to create the machines.
 
+[NOTE]
+====
+You must use the same ISO image that you used to install a cluster to deploy all new nodes in a cluster, before or after the cluster upgrades. It is recommended to use the same Ignition config file.
+The nodes automatically update themselves on the first boot before running the workloads. You can add the nodes before or after the updates.
```

Comment:
This note is using a mix of "upgrade" and "update". Just wondering if that is correct, or if "upgrade" above should be "update"

Reply:
Thanks I will keep it consistent with upgrade